CIP-?/README.md

```diff
@@ -0,0 +1,99 @@
+---
+CIP: [To be assigned]
+Title: Domain Address Resolving Standard
+Status: Proposed
+Category: Wallets
+Authors:
+  - Hinson Wong <hinson@cns.space>
+Implementors: []
+Discussions: https://github.com/cns-space/CIPs
+Solution-To: N/A
```

Comment:
```suggestion
```
This field is only required when there's a matching Cardano Problem Statement (CPS).